definitions/src/row_status.rs

```diff
@@ -0,0 +1,6 @@
+use crate::USmallInt;
+
+pub const SQL_ROW_SUCCESS: USmallInt = 0;
```

Comment:
I'm good with putting this wherever (also an enum or constant would work fine), but I'm confused on what our convention actually is haha. It seems like some things are constants, other things are enums, some things have their own file, and other things are in the `lib.rs`? But I can't tell when we should do one or the other haha.

Reply:
yeah "convention" is probably too heavy handed a word lol, mainly because we forked from odbc-sys but are now handling it ourselves. I think, in general, a good convention that we can stick with / use when adding things is probably just all groups of values should be enums, and all singular values (or things like bitmasks) should be constants.

in this case, looking at the [spec](https://github.com/microsoft/ODBC-Specification/blob/4dda95986bda5d3b55d7749315d3e5a0951c1e50/Windows/inc/sqlext.h#L1970) headers, since this is clearly a group of values I think an enum would be good. We don't need to include all the values since we're defining them anyways (like `SQL_ROW_DELETE` doesn't need to get added eg), so something like the following might be the most consistent option:
```
#[allow(non_camel_case_types)]
#[derive(Debug, PartialEq, Eq, Clone, Copy)]
#[repr(u16)]
pub enum ParamType {
    SQL_ROW_SUCCESS = 0,
    SQL_ROW_NOROW = 3,
    SQL_ROW_ERROR = 5,
    SQL_ROW_SUCCESS_WITH_INFO = 6
}
```

Reply:
Ok that makes sense haha. Yeah I agree, an enum would be good for those values.